package.json

```
@@ -42,61 +42,61 @@
   "author": "Jason Miller <jason@developit.ca> (http://jasonformat.com)",
   "license": "MIT",
   "dependencies": {
-    "@babel/core": "^7.2.2",
-    "@babel/plugin-proposal-class-properties": "7.2.1",
+    "@babel/core": "^7.4.0",
+    "@babel/plugin-proposal-class-properties": "7.4.0",
     "@babel/plugin-syntax-jsx": "^7.2.0",
-    "@babel/polyfill": "^7.0.0",
+    "@babel/polyfill": "^7.4.0",
     "asyncro": "^3.0.0",
-    "autoprefixer": "^9.0.0",
-    "babel-plugin-transform-async-to-promises": "^0.8.3",
+    "autoprefixer": "^9.5.0",
+    "babel-plugin-transform-async-to-promises": "^0.8.6",
     "babel-plugin-transform-replace-expressions": "^0.2.0",
     "brotli-size": "^0.0.3",
-    "camelcase": "^5.0.0",
-    "chalk": "^2.4.0",
-    "cssnano": "^4.1.7",
+    "camelcase": "^5.2.0",
+    "cssnano": "^4.1.10",
     "es6-promisify": "^6.0.1",
+    "filesize": "^4.1.2",
     "gzip-size": "^5.0.0",
+    "kleur": "^3.0.2",
+    "module-details-from-path": "^1.0.3",
     "pretty-bytes": "^5.1.0",
-    "rollup": "^0.67.3",
+    "rollup": "^1.7.0",
     "rollup-plugin-alias": "^1.5.1",
     "rollup-plugin-babel": "^4.1.0-0",
     "rollup-plugin-buble": "^0.19.4",
-    "rollup-plugin-bundle-size": "^1.0.1",
-    "rollup-plugin-commonjs": "^9.0.0",
+    "rollup-plugin-bundle-size": "^1.0.3",
+    "rollup-plugin-commonjs": "^9.2.1",
     "rollup-plugin-es3": "^1.1.0",
     "rollup-plugin-flow": "^1.1.1",
-    "rollup-plugin-json": "^3.1.0",
+    "rollup-plugin-json": "^4.0.0",
     "rollup-plugin-node-resolve": "^4.0.0",
-    "rollup-plugin-postcss": "^1.6.1",
-    "rollup-plugin-preserve-shebang": "^0.1.6",
-    "rollup-plugin-sizes": "^0.4.2",
-    "rollup-plugin-terser": "^3.0.0",
-    "rollup-plugin-typescript2": "^0.19.0",
+    "rollup-plugin-postcss": "^2.0.3",
+    "rollup-plugin-terser": "^4.0.4",
+    "rollup-plugin-typescript2": "^0.20.1",
     "sade": "^1.4.0",
     "tiny-glob": "^0.2.6",
     "tslib": "^1.9.0",
-    "typescript": ">=2.8.3"
+    "typescript": "^3.3.4000"
```

Now that [TypeScript 3.4](https://github.com/Microsoft/TypeScript/releases/tag/v3.4.1) is a thing, do you think we could make that read;

```suggestion
    "typescript": "^3.4.0"
```